scrapely/compat.py

```diff
@@ -0,0 +1,6 @@
+try:
+    utext = unicode
+except NameError:
+    class utext(str):
```

Comment:
Is it for doctests? scrapely uses https://pypi.python.org/pypi/doctest-ignore-unicode nose plugin to handle u prefixes in doctests; is this custom function needed?